projects/igniteui-angular/src/lib/action-strip/action-strip.component.ts

```diff
@@ -91,8 +101,8 @@ export class IgxActionStripComponent extends DisplayDensityBase {
      * @hidden
      * @internal
      */
-    @HostBinding('class.igx-action-strip')
-    public cssClass = 'igx-action-strip';
+    @Input()
+    public class = 'igx-action-strip';
```

Comment:
Wouldn't that override the `'igx-action-strip'` class if something is set on the element? I assumed that's required for the component to work. The whole point was more toward not binding the entire `attr.class` if we can avoid it, so again - can the density classes live on the internal container?

Reply:
It won't as their is additional `@HostBinding('attr.class')`. I tried applying 'igx-action-strip' without success. The styles are expecting to be applied on the host element.
And I prefer not to re-write them.
But if that is not OK, I'll restructure the strip and see if we can change the styles.

Reply:
@damyanpetev Is the current state of the classes acceptable? If not this will require re-writing some styles.